DISCOVER/07_code_of_conduct.md

```diff
@@ -69,19 +69,23 @@ Make sure that **everyone involved in your conference/event** is aware that the
 **Why You Need a Code of Conduct**
 - [Why to have a CoC and how to make one](http://incisive.nu/2014/codes-of-conduct/)
 - [Why conferences need a code of conduct](https://jacobian.org/writing/codes-of-conduct/)
-- [You literally cannot pay me to speak without a Code of Conduct](http://rachelnabors.com/2015/09/01/code-of-conduct/)
+<!--NOT AVAILABLE- [You literally cannot pay me to speak without a Code of Conduct](http://rachelnabors.com/2015/09/01/code-of-conduct/)-->
 - [Human Decency Is Not Enough: Why Cons Need Better Anti-Harassment Policies](https://www.wired.com/2013/07/convention-harassment-comic-con/)
 - [Should my tech conference community have a code of conduct and recommended resources](http://wunder.schoenaberselten.com/2016/02/17/should-my-tech-conference-community-have-a-code-of-conduct-recommended-resources/)
 
 **Crafting a Code of Conduct**
 - [A CoC builder](http://codeofconduct.io)
 - [Conference Code of Conduct](http://confcodeofconduct.com/)
 - [Codes of Conduct 101 + FAQ](https://www.ashedryden.com/blog/codes-of-conduct-101-faq)
-- [How to design a code of conduct for your community](https://adainitiative.org/2014/02/18/howto-design-a-code-of-conduct-for-your-community/)
+<!--NOT AVAILABLE
+- [How to design a code of conduct for your community](https://adainitiative.org/2014/02/18/howto-design-a-code-of-conduct-for-your-community/)-->
```

Comment:
available at: https://web.archive.org/web/20201026215352/https://adainitiative.org/2014/02/18/howto-design-a-code-of-conduct-for-your-community/